README.md

```diff
@@ -119,7 +119,7 @@ The steps below will help install and troubleshoot the report processor on a sta
 7. If configured properly the Puppet Report Viewer app in Splunk will show 1 node in the `Overview` tab.
 
 8. Now it is time to roll these settings out to the fleet of Puppet Servers in the installation. For PE users:
-  * In the PE console, navigate to `Node groups` and expand `PE Infrastructure`.
+  * In the [PE console](https://puppet.com/docs/pe/2021.2/config_console.html), navigate to `Node groups` and expand `PE Infrastructure`.
```

Comment:
Maybe this one instead: https://puppet.com/docs/pe/2021.2/console_accessing.html